crypto/perlasm/x86_64-xlate.pl

```diff
@@ -1047,6 +1047,7 @@
 					    push(@segment_stack, ".text\$");
 				        }
 					$v="$current_segment\tENDS\n" if ($current_segment);
+					$$line =~ s/([^,]*).*/$1/;
```

Comment:
I'd suggest moving this statement to line 1015.

Reply:
If this is intended to remove everything from the first comma until the end of the line, the idiomatic way to do this in Perl would be

```
$$line =~ s/,.*//;
```

Reply:
Added a commit taking in the two suggestions.